e2e/cypress/integration/messaging/local_date_time_spec.js

```diff
@@ -77,10 +77,10 @@ describe('Messaging', () => {
                 locale: 'es',
                 manualTimezone: 'NZ-CHAT',
                 localTimes: [
-                    {postIndex: 0, standard: '06:15 PM', military: '18:15'},
-                    {postIndex: 1, standard: '02:15 AM', military: '02:15'},
-                    {postIndex: 2, standard: '10:15 AM', military: '10:15'},
-                    {postIndex: 3, standard: '02:15 PM', military: '14:15'},
+                    {postIndex: 0, standard: '6:15 p. m.', military: '18:15'},
+                    {postIndex: 1, standard: '2:15 a. m.', military: '2:15'},
+                    {postIndex: 2, standard: '10:15 a. m.', military: '10:15'},
+                    {postIndex: 3, standard: '2:15 p. m.', military: '14:15'},
```

Comment:
I've ran the test and the changes here failed. Please take a look at https://mm-cypress-report.s3.amazonaws.com/663-webapp-pr-5390-1361efd/mochawesome.html

Reply:
Hmmm, okay.  These E2E tests run fine in Chrome, FF, and Edge, but not Electron.

I think there are discrepancies between CI and runtime environments regarding timezones supported by Intl.DateTimeFormat.

All current versions of Chromium/FF/Safari/Edge have full `Intl.DateTimeFormat` support for all timezones that `moment-timezone` supports. In these environments (`cypress:run`, `cypress:run:firefox`), `NZ-CHAZ` is supported and proceeds to get translated to `es`. 

In other environments (Electron, Jest, etc.) `NZ-CHAZ` and other timezones are not supported and subsequently fail, which causes the fallback `moment-timezone` formatter to kick in, which does not have translations—resulting in `en`.


What do you think, should I loosen the check here to accept both `en` and `es`? (Hopefully at least until we can unify CI/runtime `Intl` support?)

Reply:
I went ahead and switched to `en` as the test case is validating unsupported timezone and `es` is checked in the case above.

Reply:
Our CI runs in chrome 83 (headless) currently. Not yet running on Firefox 'cause there's still an issue with it, nor with Edge. I'll try to run again then see what's going on.